tests/baselines/reference/callWithMissingVoid.errors.txt

```diff
@@ -28,19 +28,19 @@ tests/cases/conformance/expressions/functionCalls/callWithMissingVoid.ts(75,1):
     
     declare const xAny: X<any>;
     xAny.f() // error, any still expects an argument
-    ~~~~~~~~
+         ~

```

Comment:
I think the idea was that the argument list was supposed to be included in the range.

Reply:
Ah, just saw your comment in the PR.